ballerina/client.bal

```diff
@@ -1,15 +1,1363 @@
-// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
-//
-// WSO2 LLC. licenses this file to you under the Apache License,
-// Version 2.0 (the "License"); you may not use this file except
-// in compliance with the License.
-// You may obtain a copy of the License at
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing,
-// software distributed under the License is distributed on an
-// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-// KIND, either express or implied.  See the License for the
-// specific language governing permissions and limitations
-// under the License.
+// AUTO-GENERATED FILE. DO NOT MODIFY.
```

Comment:
Shall we use `--license` option along with the bal openapi command to update the license headers, you can find the license header here https://github.com/ballerina-platform/module-ballerinax-intercom/blob/main/docs/license.txt

Reply:
@lnash94 Thank you for bringing this to my attention! I value your feedback and I think I have implemented the necessary adjustments

Reply:
@PasinduDilmin, I believe you have generated code by executing `bal openapi -i openapi.json --mode client -o ballerina`
As a minor suggestion regarding license headers, instead of keeping the auto-generated default one[1]

[1] Auto-generated
```
// AUTO-GENERATED FILE. DO NOT MODIFY.
// This file is auto-generated by the Ballerina OpenAPI tool.
```

Shall we update the client.bal, types.bal, and utils.bal with the below content[2], 
this content sample it can be found [here](https://github.com/ballerina-platform/module-ballerinax-intercom/blob/main/docs/license.txt)

[2]
```
// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
//
// WSO2 LLC. licenses this file to you under the Apache License,
// Version 2.0 (the "License"); you may not use this file except
// in compliance with the License.
// You may obtain a copy of the License at
//
// http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing,
// software distributed under the License is distributed on an
// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
// KIND, either express or implied.  See the License for the
// specific language governing permissions and limitations
// under the License.
```

You can update this via the below command

```
bal openapi -i docs/spec/openapi.json --mode client  --license docs/license.txt -o ballerina
```